guides/source/active_record_migrations.md

```diff
@@ -832,8 +830,8 @@ In this example, we're updating the `price` column of the products table to
 WARNING: Modifying data directly in migrations should be approached with
 caution. Consider if this is the best approach for your use case, and be aware
 of potential drawbacks such as increased complexity and maintenance overhead,
-risks to data integrity and database portability. You can read more about data
-migrations [here](#data-migrations).
+risks to data integrity and database portability. You can read more about [data
+migrations](#data-migrations).
```

Comment:
I like the phrasing of "You can read more about..." but this sentence ends abruptly now.
It's a bit wordy and doesn't flow as good, but maybe:
```suggestion
risks to data integrity and database portability. You can read the [data
migrations documentation](#data-migrations) for more details.
```

Or maybe just:
```suggestion
risks to data integrity and database portability. See the [data migrations
documentation](#data-migrations) for more details.
```

Reply:
I prefer the instructive "See..."